internal/pkg/cli/pipeline_init.go

```diff
@@ -434,13 +437,17 @@ func (url ccRepoURL) parse() (ccRepoDetails, error) {
 	case strings.HasPrefix(urlString, "codecommit::"):
 		parsedURL := strings.Split(urlString, ":")
 		region = parsedURL[2]
+	case strings.HasPrefix(urlString, "codecommit://"):
+		// Use default profile region.
```

Comment:
Would it be better to keep all region related logic in this method? i.e. move the above changes here, something like:
```go
appRegion := aws.StringValue(sess.Config.Region)
if appRegion == "" {
   return ccRepoDetails{}, errors.New("unable to find a region associated with your session")
}
region = appRegion
```

Reply:
Hmm the reason behind this is because the `ccRepoURL.parse()` method is supposed to parse info from the URL instead of getting from elsewhere.

Reply:
that makes sense!